flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/dialect/DataSourceDialect.java

```diff
@@ -79,4 +79,13 @@ public interface DataSourceDialect<C extends SourceConfig>
      */
     @Override
     default void notifyCheckpointComplete(long checkpointId) throws Exception {}
```

Comment:
It seem that this method is no longer use

Reply:
Have to provide a default implementation, as the supper.CheckpointListener does not have a default implementation.